server/services/fcm-target.js

```diff
@@ -69,7 +69,17 @@ module.exports = ({ strapi }) => ({
                 startLimit.start || 0,
                 startLimit.limit || 25)
         );
-        const rows = results.rows || results[0];
+        let rows
```

Comment:
it can be
```js
const rows = results.rows || results[0] || results;
```
without the below switch! but your code its more safe, i have nothing to say 👍

Reply:
The response is not null in sqllite case because result[0] returns the first row of the result instead of an array of results. So your option probably wouldn't work :(

Reply:
yes true